OpenRA.Mods.RA/Traits/MadTank.cs

```diff
@@ -147,7 +147,7 @@ void StartDetonationSequence()
 
 			self.World.AddFrameEndTask(w => EjectDriver());
 			if (info.ThumpSequence != null)
-				renderUnit.PlayCustomAnimRepeating(self, info.ThumpSequence);
+				renderUnit.PlayCustomAnimationRepeating(self, info.ThumpSequence);
```

Comment:
but that might be a bit too scope-creepy for this pr (we'll need to do this change on a much wider scale in a future pr).